### PR TITLE
Created 1st website using HTML and CSS 

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

**Review by nehalskdev — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          alt="RetroSyperb image"
+        />
+        <h2 class="product-name">Bicycle RetroSyperb Vii #1</h2>
+
+        <span class="price">299.99$</span>
+        <a href="#" class="add-to-bag">Add to shoping basket</a>
+        <img class="rating" src="images/stars.png" alt="star-rating" />
+      </section>
+
+      <div id="footer-banner">
```
Comment (mmusk) [inline tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/index.html:41]: you can use `footer` here @Naufil-Shemle

File: tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/style.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  font-size: 16px;
+  margin-bottom: 0;
+}
+
+#shop span.price {
+  display: block;
+  font-size: 58px;
+}
+
+#shop a.add-to-bag {
```
Comment (mmusk) [inline tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/style.css:61]: group these properties @Naufil-Shemle

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/style.css (hunk 2 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/style.css:64]: Done bhai

File: tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/index.html:41]: I just followed the tutorial, in that he used footer-banner

**Review by mmusk — APPROVED** [review]
LGTM

**Pull request merged**